tests/Umbraco.Tests.Integration/Umbraco.Examine.Lucene/UmbracoExamine/BackOfficeExamineSearcherTests.cs

```diff
@@ -80,7 +80,7 @@ private IEnumerable<ISearchResult> BackOfficeExamineSearch(string query, int pag
             out _,
             null,
             null,
```

Comment:
This should have an extra parameter for `trashed` as `null`, so it's not using an obsoleted method.